test/tests/meshgenerators/copy_element_id/tests

```diff
@@ -0,0 +1,12 @@
+[Tests]
+  [./copy_element_id_test]
+    type = 'Exodiff'
+    input = 'copy_elem_id_test.i'
+    exodiff = 'copy_elem_id_test_out.e'
+    requirement = 'The system shall include the ability to copy an element ID to other element IDs.'
+    design = 'meshgenerators/CopyElementID.md'
+    issues = '#19297'
+    mesh_mode = 'REPLICATED'
```

Comment:
```suggestion
```
hopefully dont need that

Reply:
Good point. I will double check. I do not think we need this.